source/amazon/services/prerequisites/dependencies.rst

```diff
@@ -35,22 +35,36 @@ Pip
 
 The required modules can be installed with Pip, the Python package manager. Most of UNIX distributions have this tool available in their software repositories, but it can be compiled from sources too:
 
-a) For **CentOS/RHEL/Fedora** systems:
+A) For **CentOS/RHEL/Fedora** systems:
+
+a. Using ``yum``:
 
 .. code-block:: console
 
   # yum update && yum install python3-pip
 
-b) For **Debian/Ubuntu** systems:
+b. From sources:
+
+.. code-block:: console
+
+  # yum update && yum install curl python3-distutils
+  # curl -O https://bootstrap.pypa.io/get-pip.py
+  # python3 get-pip.py
+
```

Comment:
Is this necessary? It is pretty much the same but with more steps. Feel free to discuss it here.

Reply:
After discussing it with the rest of the team we concluded we should remove this as it is no longer needed.